auth_app/templates/auth_app/faq.html

```diff
@@ -0,0 +1,115 @@
+                        <a class="collapse-item">
+                            <strong>Yes.</strong> During a tournament, players can be playing in more than one match at a time.<br>
+                            Each one will have an individual board. So that, an asynchronous code will be required.
+                        </a>
+                    </div>
+                </div>
+                <!-- Question 5 -->
+                <a class="card-header" href="#" data-toggle="collapse" data-target="#answerFive"
+                    aria-expanded="true" aria-controls="answerFive">
+                    <span>As regards EDA selection: If I win the competition, will you hire me? And if I lose, won’t you hire me?</span>
+                </a>
+                <div id="answerFive" class="collapse" aria-labelledby="headingTwo" data-parent="#FaqBar">
+                    <div class="bg-white py-2 collapse-inner rounded ml-3">
+                        <a class="collapse-item">
+                            The competition result has no relationship with the EDA selection.
+                        </a>
+                    </div>
+                </div>
+                <!-- Question 6 -->
+                <a class="card-header" href="#" data-toggle="collapse" data-target="#answerSix"
+                    aria-expanded="true" aria-controls="answerSix">
+                    <span>As regards EDA selection: What are we looking for</span>
+                </a>
+                <div id="answerSix" class="collapse" aria-labelledby="headingTwo" data-parent="#FaqBar">
+                    <div class="bg-white py-2 collapse-inner rounded ml-3">
+                        <a class="collapse-item">
+                            We are looking for efficient object-oriented or functional code.<br>
+                            Coding Tips:<br>
+                            <li>Tests</li>
+                            <li>Patterns</li>
+                            <li>Continuous integration</li>
+
+                        </a>
+                    </div>
+                </div>
+                <!-- Question 7 -->
+                <a class="card-header" href="#" data-toggle="collapse" data-target="#answerSeven"
+                    aria-expanded="true" aria-controls="answerSeven">
+                    <span>As regards EDA selection: If I use Python, will I improve my chances of being selected</span>
+                </a>
+                <div id="answerSeven" class="collapse" aria-labelledby="headingTwo" data-parent="#FaqBar">
+                    <div class="bg-white py-2 collapse-inner rounded ml-3">
+                        <p class="collapse-item">
+                            <strong>No.</strong> You can use the language you like most.<br>
+                            If you are finally hired, the EDA training includes Python.<br>
+                            We will evaluate your code or your solution no matter what language you have used.
```

Comment:
nit : I know we don´t know a lot about performance and frontend, anyway we could think about refactor this in a future?

Reply:
Yes. The correct way to solve this is by creating questions and answers in databases. Then you can display as many as you want in the template.